packages/cli/src/commands/setup/graphql/features/trustedDocuments/__tests__/trustedDocuments.test.ts

```diff
@@ -1,22 +1,22 @@
 let mockExecutedTaskTitles: Array<string> = []
 let mockSkippedTaskTitles: Array<string> = []
 
-jest.mock('fs', () => require('memfs').fs)
-jest.mock('node:fs', () => require('memfs').fs)
-jest.mock('execa')
+vi.mock('fs', () => require('memfs').fs)
```

Comment:
Can we still do `require`? Previously you used `await import` for all memfs stuff

Reply:
Nope I'll change this out. These tests are the ones that still needed a bit of work anyway so just hadn't got to rounding off the changes in this one.